### ISSUE TITLE
[dash] Add log scale checkbox in sidebar

### ISSUE THREAD

**Issue by mdbecker** [CodeForPhilly/chime#353]
Put it at the bottom of the sidebar. When checked switch the y-axis scale to log scale. Equivalent to [`altair.ScaleType`](https://altair-viz.github.io/user_guide/generated/core/altair.ScaleType.html#altair.ScaleType)`== 'log'`

**Comment by quinn-dougherty** [comment]
dash version of https://github.com/CodeForPhilly/chime/issues/22

**Comment by kmid5280** [comment]
Would like to give this a try, but I'm trying to figure out where to actually set the logarithmic value in the plotting. If I wanted to do a straight conversion, based on the docs I should be able to add a "logy" param under the "layout" param in app/services/plotting.py and set it to True. From that point implementing the checkbox should be pretty straightforward. Am I understanding correctly? https://pandas.pydata.org/pandas-docs/version/0.22.0/generated/pandas.DataFrame.plot.html

**Comment by psahgal** [comment]
I think I figured out how to do this. As mentioned in the original post, I switched the scale type on the charting library from linear to logarithmic. I also had to make some minor adjustments to the Pandas data frame passed to the chart, because it seems Altair doesn't support values of zero when using a logarithmic scale. (I guess that's understandable, since log 0 is negative infinity.) 

I still have to add the checkbox to the sidebar. I'll work on a pull request tomorrow, unless @kmid5280 made some progress and hasn't reported it...?
